tests/tests_app/core/test_lightning_api.py

```diff
@@ -446,6 +447,27 @@ def test_configure_api():
         f"http://localhost:{APP_SERVER_PORT}/api/v1/request", data=InputRequestModel(name="hello").json()
     )
     assert response.json() == {"name": "hello", "counter": 1}
+
+    with open(__file__, "rb") as f:
+        response = requests.put(
+            f"http://localhost:{APP_SERVER_PORT}/api/v1/upload_file/example.txt", files={"uploaded_file": f}
```

Comment:
We also need to test when example.txt is omitted (the filename gets taken from the object)

Reply:
I removed it. It seems it doesn't work.